### DISCUSSION TITLE
cmd not affected by v2ray

### DISCUSSION THREAD

**Discussion by Project-Charity** [v2fly/v2ray-core#2065] [Q&A] [answered]
Hey. I'm from Iran and they've blocked every mfking programming services, such as git, or composer and etc.
I can connect to my v2ray server but it doesn't affect cmd lines and i cannot use git on cmd
Is there any way to do it
Thanks

**Comment by cross-hello** [comment, marked as answer]
Make a setting for git:
`git config --global http.proxy http://proxyUsername:proxyPassword@proxy.server.com:port`
Ref: https://gist.github.com/evantoli/f8c23a37eb3558ab8765
For example,  if you listen http port to 777 , then issue this: `git config --global http.proxy http://127.0.0.1:777`
